src/reducers/fullscreenGrid.tsx

```diff
@@ -0,0 +1,31 @@
+import { Reducer } from 'redux';
+import { ActionType, createAction, getType } from 'typesafe-actions';
+
+export const actions = {
+  toggleMainSidePanel: createAction('TOGGLE_MAIN_SIDE_PANEL'),
+};
+
+export type FullscreenGridState = {
+  mainSidePanelIsExpanded: boolean;
+};
+
+export const initialState: FullscreenGridState = {
+  mainSidePanelIsExpanded: true,
```

Comment:
I don't think you have explicit coverage for this. The first test does rely on the default value of this being `true`, but that's an assumption of the test. Maybe it would be good to add a third test to check that the initial state is true? Or maybe that would amount to testing configuration? I'm not sure, tbh.

Reply:
ah correct, I'll add a test case to ensure we don't change the default value since we want to have the panels expanded by default